import-service/src/functions/product-file/index.ts

```diff
@@ -0,0 +1,14 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
+        method: 'get',
+        path: '/import',
+        cors: true,
+      },
```

Comment:
I suppose it should have been added here 
` request: {
          parameters: {
            querystrings: {
                "name": true
            }
          }
        },`